jina/peapods/runtimes/worker/__init__.py

```diff
@@ -113,26 +113,28 @@ def wait_for_ready_or_shutdown(
 
         return False
 
-    async def Call(self, msg, *args) -> Message:
+    async def Call(self, messages: List[Message], *args) -> Message:
         """
         Process they received message and return the result as a new message
 
-        :param msg: the message to process
+        :param messages: the messages to process
         :param args: additional arguments in the grpc call, ignored
         :returns: the response message
         """
-        try:
-            return self._handle(msg)
-        except RuntimeTerminated:
-            WorkerRuntime.cancel(self.is_cancel)
-        except (RuntimeError, Exception) as ex:
-            self.logger.error(
-                f'{ex!r}' + f'\n add "--quiet-error" to suppress the exception details'
-                if not self.args.quiet_error
-                else '',
-                exc_info=not self.args.quiet_error,
-            )
-            raise
+        for msg in messages:
```

Comment:
what u want is to merge the messages right?

Reply:
yes, but not in this pr. I just wanted to add the proto changes here and see that the runtime still works. It doesnt add support for merging in the runtime